WalletWasabi.Tests/UnitTests/BitcoinCore/BitcoindBinaryHashesTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using System.Threading;
+using WalletWasabi.Microservices;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.BitcoinCore
+{
+	public class BitcoindBinaryHashesTests
+	{
+		[Fact]
+		public void VerifyBitcoindBinaryChecksumHashes()
```

Comment:
I would add a comment as this one https://github.com/zkSNACKs/WalletWasabi/pull/4864/files#diff-499380c0f64809fad717b8193bb7658eebb3de13d6b9de7deafbb39d0466787aR14 to make it clear where the hashes come from. Also these hashes does not tell me what Bitcoind version is actually used.

Reply:
I got those hashes manually, there is no file to check those hashes against. https://bitcoinknots.org/files/0.20.x/0.20.0.knots20200614/SHA256SUMS.asc

Reply:
That's a bummer. But this is very important to know when verifying those hashes, so this needs to be documented. 

```csharp
/// <summary>Bitcoin Knots distributes only SHA256 checksums for installers and for zip archives and not their content, so the validity of the hashes below depends on peer review consensus.</summary>
/// <remarks>To verify a file hash, you can use, for example, <c>Get-FileHash bitcoind.exe</c>command in PowerShell.</remarks>
/// <seealso href="https://bitcoinknots.org/files/0.20.x/0.20.1.knots20200815/SHA256SUMS.asc"/>
```

Reply:
Done. I changed the command and the link because we are using Knots 0.20.0.